Yoast/wordpress-seo
# Improve the templates wrong variables warning.

Fixes #5900 

Better color contrast and add aria-live. Simplify HTML and CSS.

How to test:
- insert a wrong variable
- see the warning appear
- do the same with a screen reader running: the screen reader should announce the warning text
